docs/srt-live-transmit.md

```diff
@@ -208,36 +215,72 @@ explicitly. Note also special cases of the **host** and **port** parts
 
 - **mode**: enforce caller, listener or rendezvous mode
 - **port**: enforce the **outgoing** port (the port number that will be set in the UDP packet as a source port when sent from this host). This can be used only in **caller mode**.
 - **blocking**: sets the `SRTO_RCVSYN` for input medium or `SRTO_SNDSYN` for output medium
 - **timeout**: sets `SRTO_RCVTIMEO` for input medium or `SRTO_SNDTIMEO` for output medium
 - **adapter**: sets the adapter for listening in *listener* or *rendezvous* mode
 
-All other parameters are SRT socket options. Here are some most characteristic options:
-
-- **latency**: Sets the maximum accepted transmission latency and should be >= 2.5 times the RTT (default: 120ms; when both parties set different values, the maximum of the two is used for both)
-- **passphrase**: Sets the password for the encrypted transmission.
-- **pbkeylen**:  Crypto key len in bytes {16,24,32} Default: 16 (128-bit)
-- **tlpktdrop**: Whether to drop packets that are not delivered on time. Default is on.
-- **conntimeo**: Connection timeout (in ms). Caller default: 3000, rendezvous (x 10)
-
-For the complete list of options, please refer to the SRT header file `srt.h` and search for `SRT_SOCKOPT` enum type. Please note that the set of available options may be version dependent. All options are available under the lowercase name of the option without the `SRTO_` prefix. For example, `SRTO_PASSPHRASE` can be set using
-a **passphrase** parameter. The mapping table `srt_options` can be found in `common/socketoptions.hpp` file.
+All other parameters are SRT socket options.
+
+| URI param   | SRT Option | Description | Possible values |
+| ----------- | ---------- | ----------- | --------------- |
+| `transtype` | `SRTO_TRANSTYPE` | Transmission type | {`live`, `file`} |
+| `maxbw`     | `SRTO_MAXBW` | Bandwidth limit in bytes | [0, ..] |
+| `pbkeylen`  | `SRTO_PBKEYLEN` | Crypto key length in bytes. | {16, 24, 32} |
+| `passphrase`| `SRTO_PASSPHRASE` | Password for the encrypted transmission. | string |
+| `mss` | `SRTO_MSS` | MTU size | integer |
+| `fc` | `SRTO_FC` | Flow control window size | integer |
+| `sndbuf` | `SRTO_SNDBUF` | Sender buffer size in bytes | integer |
+| `rcvbuf` | `SRTO_RCVBUF` | Receiver buffer size in bytes | integer |
+| `linger` | `SRTO_LINGER` | Link linger value | integer |
+| `ipttl` | `SRTO_IPTTL` | Defines IP socket "time to live" option | bool |
+| `iptos` | `SRTO_IPTOS` | Defines IP socket type of service | bool |
+| `inputbw` | `SRTO_INPUTBW` | Defines input bandwidth in bytes | integer | 
+| `oheadbw` | `SRTO_OHEADBW` | limits bandwidth overhead, percents | integer |
+| `latency` | `SRTO_LATENCY` | Defines the maximum accepted transmission latency. Should be >= 2.5 times the RTT (default: 120ms; when both parties set different values, the maximum of the two is used for both). | integer |
```

Comment:
The main option to set the latency is `rcvlatency`. This option is simply a shortcut to set `rcvlatency` and `peerlatency` to the same value.